actuators/magnetorquer.py

```diff
@@ -116,7 +104,8 @@ def convert_dipole_moment_to_voltage(
     ) -> float:
         self.dipole_moment = np.clip(a=dipole_moment, a_min=-self.max_dipole_moment, a_max=self.max_dipole_moment)
         I = self.convert_dipole_moment_to_current(self.dipole_moment)
-        self.current = np.clip(a=I, a_min=-self.max_current, a_max=self.max_current)
+        # NOTE : HARDCODED a safety margin
+        self.current = np.clip(a=I, a_min=-0.6*self.max_current, a_max=0.6*self.max_current)
```

Comment:
we can have this margin defined in the config file. It should a lot smaller than this, barely worth considering even because the error will be a small percentage. Also, the max current draw will be self-imposed in the sense that we'll want to limit power consumption and wont use the magnetorquers to 100% capacity, so not respecting this limit is not critical. I come back on what I said yesterday on this. 

So I would say: its ok to disperse the parameters that will make this limit change, but it should only throw an error in the hardware if it goes over the physical limits of the magnetorquer (max current rating and max voltage), instead of max voltage self-imposed limit, which is what is limiting the torque the most. So we may want to ditch the safety margins, lower the error in this significantly (an order of magnitude or two) and allow the magnetorquers to draw a bit more power than the imposed limit